sktime/forecasting/base/adapters/_statsmodels.py

```diff
@@ -41,11 +45,7 @@ def fit(self, y, X=None, fh=None):
         # so we coerce them here to pd.RangeIndex
         if isinstance(y, pd.Series) and type(y.index) == pd.Int64Index:
```

Comment:
hm, this should perhaps be a `BaseForecaster` functionality - let's make a note of this (but no action required)